### ISSUE TITLE
[Mediaset] No video formats found!

### ISSUE THREAD

**Issue by tone87** [yt-dlp/yt-dlp#3399]
### Checklist

- [X] I'm reporting a broken site
- [X] I've verified that I'm running yt-dlp version **2022.04.08** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are alive and playable in a browser
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/ytdl-org/youtube-dl#video-url-contains-an-ampersand-and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [x] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Italy

### Description

I cannot download videos from Mediaset, not found the video formats ..

### Verbose log

```shell
[root@static ~]# yt-dlp https://www.mediasetplay.mediaset.it/video/lisoladeifamosi/jovana-djordjevic-rientra-in-gioco_F310817401007C14 --proxy "socks5://195.231.xxx.xxx:1080" --verbose
[debug] Command-line config: ['https://www.mediasetplay.mediaset.it/video/lisoladeifamosi/jovana-djordjevic-rientra-in-gioco_F310817401007C14', '--proxy', 'socks5://195.231.xxx.xxx:1080', '--verbose']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, err UTF-8, pref UTF-8
[debug] yt-dlp version 2022.04.08 [7884ade65]
[debug] Python version 3.6.8 (CPython 64bit) - Linux-3.10.0-1160.el7.x86_64-x86_64-with-centos-7.9.2009-Core
[debug] Checking exe version: ffmpeg -bsfs
[debug] Checking exe version: ffprobe -bsfs
[debug] exe versions: ffmpeg 3.4.9, ffprobe 3.4.9
[debug] Optional libraries: brotli, certifi, Cryptodome, mutagen, sqlite, websockets
[debug] Proxy map: {'http': 'socks5://195.231.xxx.xxx:1080', 'https': 'socks5://195.231.xxx.xxx:1080'}
[debug] [Mediaset] Extracting URL: https://www.mediasetplay.mediaset.it/video/lisoladeifamosi/jovana-djordjevic-rientra-in-gioco_F310817401007C14
[Mediaset] F310817401007C14: Downloading JSON metadata
[Mediaset] F310817401007C14: Downloading MPEG4 SMIL data
[Mediaset] F310817401007C14: Downloading M3U SMIL data
[Mediaset] F310817401007C14: Downloading JSON metadata
ERROR: [Mediaset] F310817401007C14: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/yt_dlp/YoutubeDL.py", line 1408, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/yt_dlp/YoutubeDL.py", line 1492, in __extract_info
    return self.process_ie_result(ie_result, download, extra_info)
  File "/usr/local/lib/python3.6/site-packages/yt_dlp/YoutubeDL.py", line 1548, in process_ie_result
    ie_result = self.process_video_result(ie_result, download=download)
  File "/usr/local/lib/python3.6/site-packages/yt_dlp/YoutubeDL.py", line 2487, in process_video_result
    self.raise_no_formats(info_dict)
  File "/usr/local/lib/python3.6/site-packages/yt_dlp/YoutubeDL.py", line 982, in raise_no_formats
    expected=has_drm or ignored or expected)
yt_dlp.utils.ExtractorError: [Mediaset] F310817401007C14: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
```

**Comment by dirkf** [comment]
yt-dl:
```ShellSession
$ python3.9 -m youtube_dl -v -F 'https://www.mediasetplay.mediaset.it/video/lisoladeifamosi/jovana-djordjevic-rientra-in-gioco_F310817401007C14'
[debug] System config: ['--prefer-ffmpeg']
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', '-F', 'https://www.mediasetplay.mediaset.it/video/lisoladeifamosi/jovana-djordjevic-rientra-in-gioco_F310817401007C14']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, pref UTF-8
[debug] youtube-dl version 2021.12.17
[debug] Git HEAD: f70619bfe
[debug] Python version 3.9.7 (CPython) - Linux-4.4.0-210-generic-i686-with-glibc2.23
[debug] exe versions: avconv 4.3, avprobe 4.3, ffmpeg 4.3, ffprobe 4.3
[debug] Proxy map: {}
[Mediaset] F310817401007C14: Downloading JSON metadata
[Mediaset] F310817401007C14: Downloading MPEG4 SD SMIL data
[Mediaset] F310817401007C14: Downloading MPEG-DASH SD SMIL data
[Mediaset] F310817401007C14: Downloading M3U SD SMIL data
[Mediaset] F310817401007C14: Downloading MPEG4 HD SMIL data
ERROR: None of the available releases match the specified AssetType, ProtectionScheme, and/or Format preferences
... [backtrace] ...
```
This may be more informative, `ProtectionScheme` especially.

**Comment by tone87** [comment]
[root@static ~]# python3 -m youtube_dl -v  -F https://www.mediasetplay.mediaset.it/video/lisoladeifamosi/jovana-djordjevic-rientra-in-gioco_F310817401007C14 --proxy "socks5://195.231.xxx.xxx:1080" --verbose
[debug] System config: ['--prefer-free-formats']
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', 'https://www.mediasetplay.mediaset.it/video/lisoladeifamosi/jovana-djordjevic-rientra-in-gioco_F310817401007C14', '--proxy', 'socks5://195.231.xxx.xxx:1080', '--verbose']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2021.06.06
[debug] Python version 3.6.8 (CPython) - Linux-3.10.0-1160.el7.x86_64-x86_64-with-centos-7.9.2009-Core
[debug] exe versions: ffmpeg 3.4.9, ffprobe 3.4.9
[debug] Proxy map: {'http': 'socks5://195.231.xxx.xxx:1080', 'https': 'socks5://195.231.xxx.xxx:1080'}
[Mediaset] F310817401007C14: Downloading JSON metadata
[Mediaset] F310817401007C14: Downloading MPEG4 SD SMIL data
[Mediaset] F310817401007C14: Downloading MPEG-DASH SD SMIL data
[Mediaset] F310817401007C14: Downloading M3U SD SMIL data
[Mediaset] F310817401007C14: Downloading MPEG4 HD SMIL data
ERROR: None of the available releases match the specified AssetType, ProtectionScheme, and/or Format preferences
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/youtube_dl/YoutubeDL.py", line 814, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/lib/python3.6/site-packages/youtube_dl/YoutubeDL.py", line 835, in __extract_info
    ie_result = ie.extract(url)
  File "/usr/lib/python3.6/site-packages/youtube_dl/extractor/common.py", line 534, in extract
    ie_result = self._real_extract(url)
  File "/usr/lib/python3.6/site-packages/youtube_dl/extractor/mediaset.py", line 157, in _real_extract
    raise first_e
  File "/usr/lib/python3.6/site-packages/youtube_dl/extractor/mediaset.py", line 147, in _real_extract
    }), guid, 'Downloading %s %s SMIL data' % (f.split('+')[0], asset_type))
  File "/usr/lib/python3.6/site-packages/youtube_dl/extractor/theplatform.py", line 52, in _extract_theplatform_smil
    error_element.attrib['abstract'], expected=True)
youtube_dl.utils.ExtractorError: None of the available releases match the specified AssetType, ProtectionScheme, and/or Format preferences

**Comment by pukkandan** [comment]
> ERROR: None of the available releases match the specified AssetType, ProtectionScheme, and/or Format preferences

This is likely DRM, but we cannot say for sure just from this message.

cc @nixxo

**Comment by pukkandan** [comment]
Can someone who has access to the content check whether it plays in browser without DRM?

**Comment by nixxo** [comment]
with yt-dpl 2022.6.29
```
❯ ydl https://mediasetinfinity.mediaset.it/video/lisoladeifamosi/jovana-djordjevic-rientra-in-gioco_F310817401007C14 -F
[Mediaset] F310817401007C14: Downloading JSON metadata
[Mediaset] F310817401007C14: Downloading MPEG4 SMIL data
[Mediaset] F310817401007C14: Checking video URL
[Mediaset] F310817401007C14: Downloading M3U SMIL data
[Mediaset] F310817401007C14: Downloading m3u8 information
[Mediaset] F310817401007C14: Downloading JSON metadata
[info] Available formats for F310817401007C14:
ID              EXT RESOLUTION |  FILESIZE   TBR PROTO | VCODEC        VBR ACODEC      ABR MORE INFO
-----------------------------------------------------------------------------------------------------------
imagestream-1-0 fs  unknown    |                 https | images                            SMIL storyboards
imagestream-1-1 fs  unknown    |                 https | images                            SMIL storyboards
hls-audio-ita   mp4 audio only |                 m3u8  | audio only        unknown         [ita] ita
hls-195         mp4 audio only | ~ 2.84MiB  195k m3u8  | audio only        mp4a.40.5  195k
http-1          mp4 unknown    |              0k https | unknown        0k unknown      0k
hls-1246        mp4 unknown    | ~18.11MiB 1246k m3u8  | avc1.4d4020 1246k video only
hls-1890        mp4 unknown    | ~27.47MiB 1890k m3u8  | avc1.4d4020 1890k video only
hls-2546        mp4 unknown    | ~37.00MiB 2546k m3u8  | avc1.4d4020 2546k video only
```
